pypit/arparse.py

```diff
@@ -3443,11 +3529,21 @@ def init(afclass, spclass):
       Class of arguments and flags
     spclass : class
       Class of spectrograph settings
+
+    Returns
+    -------
+    ftdict : dict
```

Comment:
Nothing appears to be returned by this function. Is that the intention (or perhaps I am missing something!)?